packages/kbn-unified-data-table/src/components/data_table_columns.test.tsx

```diff
@@ -351,5 +352,52 @@ describe('Data table columns', function () {
       });
       expect(gridColumns[1].schema).toBe('numeric');
     });
+
+    it('returns columns in correct format when column customisation is provided', async () => {
+      const gridColumns = getEuiGridColumns({
+        columns,
+        settings: {},
+        dataView: dataViewWithTimefieldMock,
+        defaultColumns: false,
+        isSortEnabled: true,
+        isPlainRecord: true,
+        valueToStringConverter: dataTableContextMock.valueToStringConverter,
+        rowsCount: 100,
+        services: {
+          uiSettings: servicesMock.uiSettings,
+          toastNotifications: servicesMock.toastNotifications,
+        },
+        hasEditDataViewPermission: () =>
+          servicesMock.dataViewFieldEditor.userPermissions.editIndexPattern(),
+        onFilter: () => {},
+      });
+
+      const extensionGridColumn = gridColumns[0];
+      extensionGridColumn.display = <span>test</span>;
+      const customiseColumn = {
+        extension: () => extensionGridColumn,
+      };
+
+      const customisedGridColumns = getEuiGridColumns({
```

Comment:
```suggestion
      const customizedGridColumns = getEuiGridColumns({
```
That's how we would usually spell it.

Reply:
@jughosta out here defending the Canadians 😄

Reply:
@davismcphee The moment i saw this comment, you were the 1st person who came to my mind 😆